### PR TITLE
Add python script to import storybooks

### PR DESCRIPTION
**Pull request by SnehaHS65**
Refs #174
 
#1 to import story book dataset

### CODE REVIEW

File: prepare_data.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+
+# Select environment (TEST/PROD)
+ENVIRONMENT = "TEST"
```
Comment (jo-elimu) [inline prepare_data.py:4]:
```suggestion
ENVIRONMENT = "PROD"
```

**Review by jo-elimu — APPROVED** [review]
Looks good, @SnehaHS65 🙂

It's probably better if we use the dataset from the `PROD` environment, since the data there will be more up-to-date and more accurate.

**Comment by nya-elimu** [comment]
@coderabbitai

File: env-TEST/lang-HIN/data/storybooks.csv
Comment (jo-elimu) [inline env-TEST/lang-HIN/data/storybooks.csv:1]: @SnehaHS65 With the change from `TEST` to `PROD`, this CSV file probably needs to be replaced?

**Review by jo-elimu — CHANGES_REQUESTED** [review]
Replace the folder name `env-TEST/` with `env-PROD/` (and replace the CSV file itself).

File: env-TEST/lang-HIN/data/storybooks.csv
Comment (SnehaHS65) [reply env-TEST/lang-HIN/data/storybooks.csv:1]: @jo-elimu , I took the dataset from https://github.com/elimu-ai/webapp/blob/main/src/main/resources/db/content_PROD/hin/storybooks.csv.
Isn't this the prod one?

Comment (jo-elimu) [reply env-TEST/lang-HIN/data/storybooks.csv:1]: Yes, that's right. But the file was stores within a `TEST` folder in this repo. And I see that was fixed now, in the latest commit 👍

**Review by jo-elimu — APPROVED** [review]

**Pull request merged**